docs/reference/js/api.md

```diff
@@ -29,10 +32,18 @@ If you are writing an application that needs to access the IPFS network, use `ip
 
 :::
 
-### HTTP client
+### HTTP clients
+
+[Kubo HTTP RPC API](https://www.npmjs.com/package/kubo-rpc-client) is a client library that controls an active Kubo node running through its [RPC API](../kubo/rpc.md).
+  - Use this library to talk to a Kubo node from your JavaScript application
+  - You can also interact with the [RPC API](../kubo/rpc.md) directly using `fetch()` in a browser or a module like `request` in Node.js, but using this library can be much more convenient.
+
+::: warning Changes to IPFS HTTP RPC API
 
-[JS-IPFS HTTP RPC API](https://www.npmjs.com/package/ipfs-http-client) is a client library that controls an active IPFS node (Kubo or JS-IPFS) running through its [RPC API](../kubo/rpc.md).
-  - JS-IPFS will internally use this library if it detects another node is running on your machine. You can also interact with the [RPC API](../kubo/rpc.md) directly using `fetch()` in a browser or a module like `request` in Node.js, but using this library can be much more convenient.
+There is ongoing work that will result in HTTP RPC API divergence in the future. See https://github.com/ipfs/kubo/issues/9125 for more information.
```

Comment:
@SgtPooki looks like you forgot to close `::: warning` here?

Reply:
good catch, addressed.